indico/web/templates/_session_bar.html

```diff
@@ -6,169 +6,182 @@
-                    </fieldset>
-                    <label id="tz-custom-field">
-                        <span>{% trans %}Select a custom timezone{% endtrans %}</span>
-                        <select name="tz" size="12">
-                            {% for tz in timezone_data.timezones -%}
-                                {% set is_profile_timezone = tz == profile_timezone %}
-                                <option value="{{ tz }}" {{ 'selected' if tz == session.tzinfo.zone }} {{ 'data-profile' if is_profile_timezone }}>
-                                    {% if is_profile_timezone -%}
-                                        {% trans %}{{ tz }} (your profile timezone){% endtrans %}
-                                    {%- else -%}
-                                        {{- tz -}}
-                                    {%- endif %}
-                                </option>
-                            {%- endfor %}
-                        </select>
-                    </label>
-                    {% if session.user %}
-                        <label id="tz-remember-setting">
-                            <input type="checkbox" name="update_user" value="1">
-                            <span>{%- trans %}Remember these settings{% endtrans -%}</span>
-                        </label>
-                    {% endif %}
-                    <button data-href="{{ url_for('core.change_tz') }}"
-                            data-method="POST"
-                            data-params-selector="#tz-mode-form [name=tz_mode]:checked,
-                                                  #tz-mode-form [name=tz],
-                                                  #tz-mode-form [name=update_user]:checked"
-                            data-reload-after
-                            data-ajax>
-                        {% trans %}Save{% endtrans %}
-                    </button>
-                </form>
-            </div>
-        </article>
+        <dialog data-dialog>
+            {% if timezone_data.disabled %}
+                <span class="icon-time disabled" aria-label="{% trans %}The active timezone is {{ active_tz_label }} and it cannot be changed{% endtrans %}">
+                    {{- active_tz_label -}}
+                </span>
+            {% else %}
+                {% set profile_timezone = session.user.settings.get('timezone') if session.user else '' %}
+                <article id="tz-selector-widget" aria-labelledby="tz-selector">
+                    <div>
+                        <h2 id="tz-selector">{% trans %}Choose timezone{% endtrans %}</h2>
 
-        <button id="tz-selector-link" class="i-button icon-time arrow">{{ timezone_data.active_tz }}</button>
-    {% endif %}
+                        <div id="tz-profile-setting">
+                            <span>{% trans %}Your profile timezone:{% endtrans %}</span>
+                            <span>{{ profile_timezone }}</span>
+                        </div>
```

Comment:
Something I noticed: The `{{ profile_timezone }}` is empty for users who aren't logged-in (which is fine), but it does make things look a bit weird (see below).

![image](https://github.com/user-attachments/assets/7528f05e-32b2-4ff0-8c20-82cfe0a35215)

Maybe we could just hide this section if there isn't a profile timezone?